.github/ISSUE_TEMPLATE/support-create-a-new-environment.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 'Support: Create a New Environment'
```

Comment:
Suggestion to clarify the purpose already in the title:
- "Create new consortium demo environment"